test/functional/plugin/lsp_spec.lua

```diff
@@ -2222,6 +2237,31 @@ describe('LSP', function()
 
       os.remove(new_dir)
     end)
+    it('Kills old directory buffers after renaming a directory', function()
```

Comment:
can this be merged into the `Can rename a directory` test? Don't think we need separate test cases.